logstash-core/src/main/java/org/logstash/ext/JrubyEventExtLibrary.java

```diff
@@ -87,7 +87,9 @@ public Event getEvent() {
         public IRubyObject ruby_initialize(ThreadContext context, IRubyObject[] args) {
             final IRubyObject data = args.length > 0 ? args[0] : null;
             if (data instanceof RubyHash) {
-                this.event = new Event(ConvertedMap.newFromRubyHash((RubyHash) data));
+                this.event = new Event(
+                    ConvertedMap.newFromRubyHash(context, (RubyHash) data)
```

Comment:
@jakelandis it's the same, but `o.getRuntime().getCurrentContext()` is a thread local soft reference that is populated lazily, so it's much slower to access and JRuby Javadoc recommends not doing it.